src/main/resources/lang/english.lang

```diff
@@ -1718,6 +1718,16 @@ genes:
 	weak: weak
 	aggressive: aggressive, savage, wild
 
+# -- Villager Types --
+villager types:
+	desert: desert
+	jungle: jungle
+	plains: plains
```

Comment:
I am afraid this will conflict with existing biomes, can this be confirmed?

Reply:
Appears to be no issue
<img width="641" alt="Screen Shot 2020-07-27 at 7 13 23 PM" src="https://user-images.githubusercontent.com/20008482/88611151-46585d00-d03d-11ea-8c84-629fd00f13c0.png">

Reply:
What if you use variables in place of desert, jungle etc?

Reply:
In that case there seems to be a small issue, as it overrides biomes.

But (see pic) its also already an issue with jungle for example, it tries to set it to a jungle tree

<img width="913" alt="Screen Shot 2020-07-27 at 7 20 48 PM" src="https://user-images.githubusercontent.com/20008482/88611657-59b7f800-d03e-11ea-863f-3d7c92d3b5c5.png">

Reply:
Just in case, I would add a converter VillagerType -> Biome and a comparator VillagerType <-> Biome, to avoid future issues.

Reply:
Okie dokie. I have never done either of those two, so im going to spend a little time figuring that out.

Reply:
Ok converter and comparator has been added!